From 3f22787ff3efd50fc69ad9bb5d43c99d7fa5669e Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Tue, 28 May 2024 14:33:44 +0530
Subject: [PATCH] Blissify: tweak default lock screen clock size

the default was too big for 1600x720
change it here to be a bit closer to AOSP default

this should have been user tunable anyways though...

Change-Id: I33ca588cc50ba7b18c5af92a815857268f03b1db
Signed-off-by: Sushrut1101 <guptasushrut@gmail.com>
---
 res/xml/bliss_settings_lockscreen_clock.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/xml/bliss_settings_lockscreen_clock.xml b/res/xml/bliss_settings_lockscreen_clock.xml
index 79d23e1..f59cd52 100644
--- a/res/xml/bliss_settings_lockscreen_clock.xml
+++ b/res/xml/bliss_settings_lockscreen_clock.xml
@@ -24,7 +24,7 @@
         android:max="500"
         settings:min="80"
         settings:interval="1"
-        android:defaultValue="280"
+        android:defaultValue="125"
         settings:units="dp" />
 
     <com.bliss.support.preferences.SystemSettingSeekBarPreference
@@ -34,7 +34,7 @@
         android:max="100"
         settings:min="25"
         settings:interval="1"
-        android:defaultValue="86"
+        android:defaultValue="65"
         settings:units="dp" />
 
     <com.bliss.support.preferences.SystemSettingSeekBarPreference
@@ -44,7 +44,7 @@
         android:max="180"
         settings:min="25"
         settings:interval="1"
-        android:defaultValue="180"
+        android:defaultValue="120"
         settings:units="dp" />
 
     <com.bliss.support.preferences.SystemSettingListPreference
-- 
2.44.0

